include/battle/move_display_info.h

```diff
@@ -0,0 +1,10 @@
+#ifndef POKEPLATINUM_MOVE_DISPLAY_INFO
+#define POKEPLATINUM_MOVE_DISPLAY_INFO
+
+typedef struct MoveDisplayInfo {
+    u16 move[4];
```

Comment:
**polish**: Add `#include "constants/moves.h"` at the top so that you can use `LEARNED_MOVES_MAX` in these array declarations.

Reply:
good suggestion, I've added that